cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -129,14 +109,51 @@ func (p *nfsProvisioner) Provision(ctx context.Context, options controller.Provi
 				NFS: &v1.NFSVolumeSource{
 					Server:   p.server,
 					Path:     path,
-					ReadOnly: false,
+					ReadOnly: false, // Pass ReadOnly through if in alias mode?
```

Comment:
afaik it have no real effect, you can try to set it to true and it will be writable anyway.